src/OpenSage.Game/Logic/Object/ObjectStatus.cs

```diff
@@ -57,6 +57,9 @@ public enum ObjectStatus
         [IniEnum("STATUS_RIDER7")]
         StatusRider7,
 
+        [IniEnum("IS_BRAKING")]
+        IsBraking,
```

Comment:
I don't think this can go here - at least not without moving some other things.

Somewhere in the gameobject, ObjectStatus is stored as a uint32 flags enum. We currently have 32 items in this enum, so that's a separate issue, but also this goes between `Hijacked = 9` and `Sold = 18` and the gap between those is already filled with other enums (though their values haven't been confirmed).

https://github.com/OpenSAGE/OpenSAGE/blob/e6cdafabe06f2f7dcc2e430da0fc8782560354fa/src/OpenSage.Game/Logic/Object/GameObject.cs#L1493

If this is added here, then `CanAttack` and `Sold` would share the same ordinal. We should probably take a closer look at this at some point. It might be nice to try to create more of these states to validate their values.

Reply:
That's actually why I put it near the end, close to the values that were added in BFME and later games. It's only older save files (specifically Generals) that persist the status as a uint, so the numeric values of values added in later games don't matter. But you're right, I didn't put it quite low enough - I'll move it below `AirborneTarget`.

Reply:
I've moved the values around - https://github.com/OpenSAGE/OpenSAGE/commit/64c9a55423ae005bf1e5332f832ab06bc5050df9

Reply:
Still conflicts with `InsideGarrison = 21` 😅 

That should be the last item with a known ID